packages/core/src/models/lessonPlans.ts

```diff
@@ -193,7 +189,7 @@ export class LessonPlans {
     ${summary.keywords.join(",")}.`
       : "There is no summary for this lesson";
 
-    const userPrompt = `I would like to generate a lesson plan for the lesson titled ${lesson.title} for a lesson titled "${lesson.keyStage?.title ?? "Untitled"}" in ${lesson.subject?.title ?? "an unknown subject"}. 
+    const userPrompt = `I would like to generate a lesson plan for a lesson titled ${lesson.title} in Key Stage "${lesson.keyStage?.title ?? "Untitled"}" ${lesson.subject?.title ?? "in an unknown subject"}. 
```

Comment:
This could read "in Key Stage Untitled"

Reply:
I think this might become unused. This is the original generation script for lesson plans and I think we should be using the package now instead